reporters/index.js

```diff
@@ -11,8 +11,11 @@ const loadInternalReporter = function (name) {
 };
 
 const loadExternalReporter = function (name) {
-
   try {
+    if(name.includes('@')) {
```

Comment:
`if (name.startsWith('@')) {`

space before paren to fit our coding style, and startsWith is more appropriate than includes here